File: C_Programming/Practise_Programs/PP_113.c
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_113
//
//  Description       : This program accepts a string from the user
//                      and displays the first three characters using
//                      a character pointer.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 14/08/2026
//
//  Time Complexity   : O(1)
//  Space Complexity  : O(n)
//
//////////////////////////////////////////////////////////////////

#include<stdio.h>

//////////////////////////////////////////////////////////////////
//
//  Function Name     : Display()
//
//  Description       : It is used to display the first three
//                      characters of the given string using a
//                      character pointer.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 14/08/2026
//
//////////////////////////////////////////////////////////////////

void Display(char *str)
{
    printf("%c\n", *str);
    str++;

    printf("%c\n", *str);
    str++;

    printf("%c\n", *str);
    str++;
}

int main()
{
    char Arr[50] = {'\0'};

    printf("Enter string : \n");
    scanf("%[^'\n']s", Arr);  
    
    Display(Arr);
    
    return 0;
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input :
//      Ganesh
//
//  Sample Output :
//
//      Enter string :
//      G
//      a
//      n
//
//////////////////////////////////////////////////////////////////

File: Java_Programming/Practise_Programs/PP_113.java
//////////////////////////////////////////////////////////////////
//
//  File Name         : PP_113
//
//  Description       : This program accepts a string from the user
//                      and displays the entered string.
//
//  Author            : Shubham Somanath Gadhe
//  Date              : 26/08/2026
//
//  Time Complexity   : O(n)
//  Space Complexity  : O(n)
//
//////////////////////////////////////////////////////////////////

import java.util.Scanner;

class PP_113
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);
        String str = null;

        System.out.println("Enter String : ");
        str = sobj.nextLine();

        System.out.println("String is : " + str);

        sobj.close();
    }
}

//////////////////////////////////////////////////////////////////
//
//  Sample Input :
//      Ganesh
//
//  Sample Output :
//
//      Enter String :
//      String is : Ganesh
//
//////////////////////////////////////////////////////////////////